cluster-up/cluster/kind/podpreset.sh

```diff
@@ -0,0 +1,56 @@
+ #!/usr/bin/env bash
+
+set -e
+
+source ${KUBEVIRTCI_PATH}/cluster/kind/common.sh
+
+function podpreset::enable_admission_plugin() {
+    local -r cluster_name=$1
+
+    docker exec "$cluster_name-control-plane" bash -c 'sed -i \
+    -e "s/NodeRestriction/NodeRestriction,PodPreset/" \
+    -e "/NodeRestriction,PodPreset/ a\    - --runtime-config=settings.k8s.io/v1alpha1=true" \
+    /etc/kubernetes/manifests/kube-apiserver.yaml'
+}
+
+function podpreset::validate_admission_plugin_is_enabled() {
+    local -r wait_time=$1
+    local -r admission_plugin=$2
```

Comment:
Per @qinqon [comment](https://github.com/kubevirt/kubevirtci/pull/570/files#r613772514), this needs to be hardcoded as we are in the "podpreset" context.

Reply:
Done